$OpenBSD$

Fix devel/gtest include path;  this seems impossible using MAKE_FLAGS

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -127,6 +127,7 @@ GTEST_INCLUDES += \
     -I$(SRC_PATH)gtest/googletest/include
 
 CODEC_UNITTEST_INCLUDES += \
+    -I/usr/local/include \
     -I$(SRC_PATH)gtest/googletest/include \
     -I$(SRC_PATH)codec/common/inc \
     -I$(SRC_PATH)test
